codegen/src/main/ops/org/nd4j/codegen/ops/Linalg.kt

```diff
@@ -0,0 +1,112 @@
+package org.nd4j.codegen.ops
+
+import org.nd4j.codegen.api.DataType
+import org.nd4j.codegen.api.Language
+import org.nd4j.codegen.api.doc.DocScope
+import org.nd4j.codegen.dsl.*
+
+fun Linalg() =  Namespace("Linalg") {
+    //val namespaceJavaPackage = "org.nd4j.linalg"
+
+    Op("Cholesky") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.transforms"
+        javaOpClass = "Cholesky"
+        Input(DataType.NDARRAY, "input") { description = "Input tensor with inner-most 2 dimensions forming square matrices" }
+        Output(DataType.NDARRAY, "output"){ description = "Transformed tensor" }
```

Comment:
From op definition:
* input should be numerical type, not NDArray type
* output should be floating point type

Reply:
Fixed.